src/Tizen.NUI.Components/Style/DefaultLinearItemStyle.cs

```diff
@@ -33,7 +33,7 @@ public class DefaultLinearItemStyle : RecyclerViewItemStyle
         /// <summary>
         /// Creates a new instance of a DefaultLinearItemStyle.
         /// </summary>
-        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
this line look like hiding public API, but the class is hidden already so it's ok